pkg/cli/state/vmclarity_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package state
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/openclarity/kubeclarity/shared/pkg/config"
+
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/pkg/shared/families"
+	"github.com/openclarity/vmclarity/pkg/shared/families/sbom"
+	"github.com/openclarity/vmclarity/pkg/shared/families/types"
+	"github.com/openclarity/vmclarity/pkg/shared/utils"
+)
+
+var effectiveScanConfigJSON = `{"sbom":{"Enabled":true,"AnalyzersList":["syft"],"Inputs":[{"StripPathFromResult":null,"Input":"test","InputType":"dir"}],"MergeWith":null,"AnalyzersConfig":{"Registry":null,"Analyzer":{"OutputFormat":"","AnalyzerList":["syft"],"Scope":"","TrivyConfig":{"Timeout":0,"CacheDir":"","TempDir":""}},"Scanner":null,"LocalImageScan":false}},"vulnerabilities":{"Enabled":false,"ScannersList":null,"Inputs":null,"InputFromSbom":false,"ScannersConfig":null},"secrets":{"Enabled":false,"ScannersList":null,"StripInputPaths":false,"Inputs":null,"ScannersConfig":null},"rootkits":{"Enabled":false,"ScannersList":null,"StripInputPaths":false,"Inputs":null,"ScannersConfig":null},"malware":{"Enabled":false,"ScannersList":null,"StripInputPaths":false,"Inputs":null,"ScannersConfig":null},"misconfiguration":{"enabled":false,"ScannersList":null,"StripInputPaths":false,"Inputs":null,"ScannersConfig":{"Lynis":{"InstallPath":""}}},"infofinder":{"enabled":false,"ScannersList":null,"StripInputPaths":false,"Inputs":null,"ScannersConfig":{"SSHTopology":{}}},"exploits":{"enabled":false,"ScannersList":null,"Inputs":null,"InputFromVuln":false,"ScannersConfig":null}}`
```

Comment:
Could you please use `embed` to access test data in tests. Like we did here: https://github.com/openclarity/vmclarity/blob/main/pkg/orchestrator/provider/cloudinit/cloudinit_test.go

Reply:
https://github.com/openclarity/vmclarity/pull/739/commits/35ee574999f8c1d3e31278c25d3ed409f4099feb